src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -117,12 +117,18 @@ public static int parsePositiveInteger(String num) throws IllegalValueException
      * Parses three {@code String facebook, twitter, instagram} into an {@code SocialMedia}.
      */
     public static SocialMedia parseSocialMedia(Optional<String> facebook,
-            Optional<String> twitter, Optional<String> instagram) throws IllegalValueException {
+            Optional<String> twitter, Optional<String> instagram, boolean isAdding) throws IllegalValueException {
         requireAllNonNull(facebook, twitter, instagram);
 
```

Comment:
Hi @kenpaxtonlim , when will IllegalValueException be thrown?

Reply:
Oh I forget to delete that, previously the constructor will throw it but that was removed.